### ISSUE TITLE
Does not appear to install by default on python 3.6

### ISSUE THREAD

**Issue by antocuni** [pdbpp/pdbpp#64]
Originally reported by: **Anonymous**

----------------------------------------

Collecting pdbpp
  Downloading pdbpp-0.8.3.tar.gz (40kB)
    100% |################################| 40kB 1.3MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-3cnupscy/pdbpp/setup.py", line 12, in <module>
        long_description = open(readme).read() + '\n\n' + open(changelog).read()
      File "/root/.local/envs/glfw-cffi/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 591: ordinal not in range(128)

    ----------------------------------------

----------------------------------------
- Bitbucket: https://bitbucket.org/antocuni/pdb/issue/64

**Comment by antocuni** [comment]
*Original comment by* **Brian Bruggeman (Bitbucket: [bbruggeman](https://bitbucket.org/bbruggeman), GitHub: Unknown)**:

----------------------------------------

```

Collecting pdbpp
  Downloading pdbpp-0.8.3.tar.gz (40kB)
    100% |################################| 40kB 1.3MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-3cnupscy/pdbpp/setup.py", line 12, in <module>
        long_description = open(readme).read() + '\n\n' + open(changelog).read()
      File "/root/.local/envs/glfw-cffi/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 591: ordinal not in range(128)

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-3cnupscy/pdbpp/

```

**Comment by antocuni** [comment]
*Original comment by* **Aleks Bunin (Bitbucket: [b-enlnt](https://bitbucket.org/b-enlnt), GitHub: Unknown)**:

----------------------------------------

This is duplicate of #46.

**Comment by antocuni** [comment]
*Original comment by* **Aleks Bunin (Bitbucket: [b-enlnt](https://bitbucket.org/b-enlnt), GitHub: Unknown)**:

----------------------------------------

pull request #48 should resolve this issue.

**Comment by antocuni** [comment]
*Original comment by* **Antonio Cuni (Bitbucket: [antocuni](https://bitbucket.org/antocuni), GitHub: [antocuni](https://github.com/antocuni))**:

----------------------------------------

Fix setup on Python 3.x.

Fixes #46.
Fixes #64.
